Aztec/Classes/TextKit/TextView.swift

```diff
@@ -555,7 +555,11 @@ open class TextView: UITextView {
     // MARK: - Pasteboard Helpers
 
     internal func storeInPasteboard(encoded data: Data, pasteboard: UIPasteboard = UIPasteboard.general) {
-        pasteboard.setData(data, forPasteboardType: NSAttributedString.pastesboardUTI)
+        if pasteboard.numberOfItems > 0 {
+            pasteboard.items[0][NSAttributedString.pastesboardUTI] = data;
```

Comment:
I don't understand why we're doing this one. Why not use [setItems(_:options:)](https://developer.apple.com/documentation/uikit/uipasteboard/1829417-setitems) in both cases?

Reply:
I tried that, but setItems will reset everything, so unless you first copy what you already have there and add the extra UTI to it.